fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/mip.xml

```diff
@@ -89,27 +104,19 @@
             <!-- RTO ARM pb -->
             <UseTemplate Name="FBW_Push_Toggle">
                 <NODE_ID>PUSH_RTO_ARM</NODE_ID>
-                <TOOLTIPID>ARM RTO</TOOLTIPID>
+                <TOOLTIPID>TT:A380X.COCKPIT.TOOLTIPS.AUTOBRAKE_RTO</TOOLTIPID>
                 <DISABLE_SEQ1 />
                 <MOMENTARY />
-                <SEQ2_CODE>(L:A32NX_AUTOBRAKES_ARMED_MODE, Number) 3 ==</SEQ2_CODE>
-                <LEFT_SINGLE_CODE>1 (&gt;L:A32NX_OVHD_AUTOBRK_MAX_ON_IS_PRESSED)</LEFT_SINGLE_CODE>
-                <LEFT_LEAVE_CODE>0 (&gt;L:A32NX_OVHD_AUTOBRK_MAX_ON_IS_PRESSED)</LEFT_LEAVE_CODE>
+                <SEQ2_CODE>(L:A32NX_AUTOBRAKES_RTO_ARMED)</SEQ2_CODE>
+                <LEFT_SINGLE_CODE>1 (&gt;L:A32NX_OVHD_AUTOBRK_RTO_ARM_IS_PRESSED)</LEFT_SINGLE_CODE>
```

Comment:
I think this should be a momentary as it does not stay "in" (it is not a switch) - so it either resets itself to 0 or the systems code should. 
Otherwise after pressing it via API the user must reset it again so it is not "held" all the time.

Reply:
Working as expected indeed - resolved.
https://github.com/flybywiresim/aircraft/assets/16833201/07e17c32-9b21-4352-916a-31947d8c16cb